modules/Layout.js

```diff
@@ -246,6 +231,8 @@ Layout.prototype.render = function (l) {
         g.setFont(l.font).setFontAlign(0,0,l.r).drawString(l.label, l.x+(l.w>>1), l.y+(l.h>>1));
       }
     }, "btn":function(l){
+      if (l.font && l.font.endsWith("%"))
```

Comment:
I don't think this code should be here. `update` gets called first which checks for `%` andoverwrites `font`, so this code should never get called.

Reply:
Thank you, I hadn't twigged that, removed as requested.